src/model/Surface.cpp

```diff
@@ -1440,16 +1440,101 @@ namespace model {
         return result;
       }
 
-      double windowArea = 0.0;
+      //double windowArea = 0.0;
+      //for (const SubSurface& subSurface : this->subSurfaces()) {
+      //  if (istringEqual(subSurface.subSurfaceType(), "FixedWindow") || istringEqual(subSurface.subSurfaceType(), "OperableWindow")) {
+      //    windowArea += subSurface.multiplier() * subSurface.roughOpeningArea();
+      //  }
+      //}
+
+      double roughOpeningArea = totalAreaOfSubSurfaces();
+      double wwr = roughOpeningArea / grossArea;
+
+      return wwr;
+    }
+
+    // Calcuklates and returns the toital area of the subsurfaces
+    // If any subsrface extends outside the parent surface or overlaps an adjacent subsurface
+    // then that subsurface's vertices rather than rough opening vertioces are used to calculate the area
+    // NOTE: I'm starting to think a better way to do this would be to use polygon booleans
+    // 1 - Intersect the subsurface rough opening with the parent surface to get the area inside the parent surface
+    // 1 - Add all the subsurface rough openings together (so overlap areas dont count twice)
```

Comment:
```suggestion
    // Calcuklates and returns the toital area of the subsurfaces
    // If any subusrface extends outside the parent surface or overlaps an adjacent subsurface
    // then that subsurface's vertices rather than rough opening vertices are used to calculate the area
    // NOTE: I'm starting to think a better way to do this would be to use polygon booleans
    // 1 - Intersect the subsurface rough opening with the parent surface to get the area inside the parent surface
    // 2 - Add all the subsurface rough openings together (so overlap areas don't count twice)
```